sigs.yaml

```diff
@@ -3245,6 +3245,44 @@ workinggroups:
     liaison:
       github: cblecker
       name: Christoph Blecker
+- dir: wg-long-term-support
+  name: Long Term Support
+  mission_statement: >
+    Define LTS
+
+  stakeholder_sigs:
+  - Architecture
+  - Cluster Lifecycle
+  - K8s Infra
+  - Release
+  - Testing
+  label: long-term-support
```

Comment:
nit, `lts` everywhere is easier to remember (and matches all the things we used before, xref 0e9d714c7657733cebe94bb9e975cb7c121349c4)

Reply:
I actually wanted to leave the same comment but removed it before submitting the review. 😅 I'm not sure if `lts` is too short, but I general `lts` might be better.

Reply:
will do!